docs/guides/push-work-pools.md

```diff
@@ -14,13 +15,14 @@ Push [work pools](/concepts/work-pools/#work-pool-overview) are a special type o
 
 In this guide you will:
 
-- Create a push work pool that sends work to Google Cloud Run or AWS ECS
+- Create a push work pool that sends work to Google Cloud Run, AWS ECS or Azure
```

Comment:
Thoughts on Azure ACI - ? It keeps it in line with "Provider -> Offering" for the Google / AWS examples

Reply:
Yeah good call out